admin/modules/cookie-policy-generator/templates/ccpa-california/de.md

```
@@ -0,0 +1,49 @@
+
+Right to Know, Delete, Correct, Opt-Out, Limit, Non-Discrimination, Portabilität. E-Mail **{{COMPANY_EMAIL}}** — Antwort innerhalb von 45 Tagen.
+
+## Global Privacy Control
+
+Wir respektieren das GPC-Signal als gültiges Opt-out.
+
+## Behörde
+
+California Privacy Protection Agency: {{CA_PIPC_CONTACT}}, [{{OFFICIAL_RESOURCES_URL}}]({{OFFICIAL_RESOURCES_URL}})
+
+## Kontakt
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)